CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,125 @@
+# GeoPandas Project Code of Conduct
+
+Behind GeoPandas Project is an engaged and respectful community made up of people
```

Comment:
```suggestion
Behind the GeoPandas Project is an engaged and respectful community made up of people
```

Does that sound better? (if so, there are a few other occurences in the text) 

Otherwise we could also use "GeoPandas" instead of "GeoPandas Project", then there is less need for a "the" I think.

Reply:
If there is something in the English language I never mastered, it would be the placement of articles :D. Your _suggestion _sounds_ better.

I was not sure which version to use, if GeoPandas or GeoPandas Project. I started with former then changed to latter. What do you prefer?

Reply:
I think "the GeoPandas Project" is fine